### PR TITLE
TF Migration 1.x >> TF 2.x (a_tfstart.ipynb) Notebook Fix

### PR DESCRIPTION
**Pull request by kalyanipande-tudip**
- Updated all the dependencies according to tensorflow 2.X requirements.
- Added the command to check version of Tensorflow installed in notebook.
- Related Buganizer: http://b/158454500

### CODE REVIEW

File: courses/machine_learning/tensorflow/a_tfstart.ipynb [hunk 1]
```diff
@@ -4,29 +4,39 @@
    "cell_type": "markdown",
```
Comment (gstripling) [inline courses/machine_learning/tensorflow/a_tfstart.ipynb:4]: <p>Please change Tensorflow to TensorFlow in the title and in the sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1015/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (gstripling) [inline courses/machine_learning/tensorflow/a_tfstart.ipynb:4]: <p>Change Tensorflow to TensorFlow in the comment line #1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1015/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gstripling) [inline courses/machine_learning/tensorflow/a_tfstart.ipynb:4]: <p>Please change Tensorflow to TensorFlow in the title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1015/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Comment by kalyanipande-tudip** [comment]
Hi @gstripling,

I have updated the above mentioned changes in the notebook. The PR can be merged now.

Feel free to let me know if there is anything else for me.

-Thanks

**Pull request merged**